resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClientSchedulerBackend.scala

```diff
@@ -164,7 +164,7 @@ private[spark] class YarnClientSchedulerBackend(
 
     super.stop()
     client.stop()
-    logInfo("Stopped")
+    logInfo("YarnClientSchedulerBackend Stopped")
```

Comment:
Isn't `YarnClientSchedulerBackend` added to full of log message, as in most case the format contains the logger name (mostly same as class name)?

Reply:
Current log message is 

> 2019-12-27 20:23:26,613 | INFO  | [main] | Stopped | org.apache.spark.internal.Logging$class.logInfo(Logging.scala:54)

Reply:
You must be using a custom logging file because by default you get:

19/12/31 10:47:28 INFO YarnClientSchedulerBackend: Stopped

Reply:
Look at the spark conf/log4j.properties.template file for the format used there

Reply:
If true, and I suspect you're right, is there much harm in just adding to the log message anyway?

Reply:
Also on other places we are using descriptive log message. Like,
https://github.com/apache/spark/blob/ce7a49f7fa30b4f252af09f9428cffc1055ef7c7/core/src/main/scala/org/apache/spark/deploy/ExternalShuffleService.scala#L169-L171

Reply:
no harm, I'm ok with the change, there may just be other places in the code that aren't clear without it so you may want to look at your logging config anyway.

Reply:
Same as @tgravescs, I think log4j format matters, but agree no harm with this.

Reply:
I don't think we should necessarily add it.

> because by default you get:
>
> 19/12/31 10:47:28 INFO YarnClientSchedulerBackend: Stopped

Also, by default, seems showing the logs fine. Otherwise, we should fix so many places as @tgravescs pointed out. If class names should be shown, that should properly configured for such places.

Reply:
Yes. as @tgravescs pointed out, here logs can be improved by changing log4j conversion pattern.